.github/renovate.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": [
+    "config:base",
+    ":disableDependencyDashboard"
+  ],
+  "timezone": "America/Los_Angeles",
+  "schedule": ["after 1am and before 7am every monday"],
+  "packageRules": [
+    {
+      "matchDepTypes": ["devDependencies"],
+      "matchUpdateTypes": ["patch", "minor", "pin", "digest"],
+      "groupName": "devDependencies (non-major)",
+      "automerge": true
```

Comment:
What does automerge do?

Reply:
Patch and minor version updates to dev dependencies will be grouped together into a single PR and, after the CI passes, merged automatically. Without `automerge` a maintainer would need to do the merge.